code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -88,7 +93,7 @@
 /obj/item/encryptionkey/heads/captain
 	name = "Captain's Encryption Key"
 	icon_state = "cap_cypherkey"
-	channels = list("Command" = 1, "Security" = 1, "Engineering" = 0, "Science" = 0, "Medical" = 0, "Supply" = 0, "Service" = 0)
+	channels = list("Command" = 1, "Security" = 1, "Engineering" = 0, "Science" = 0, "Medical" = 0, "Supply" = 0, "Service" = 0, "Oversight" = 0)
```

Comment:
Shouldn't the captain have this on by default?

Reply:
Unsure on that point, the channel was more for NT Rep, IAA, and magistrate in my mind with the option of the captain being involved. Still I have no real opinion on this point and will bow to the consensus.

Reply:
It is now on by default for the captain.